README.md

```diff
@@ -16,38 +16,74 @@
+|Intel(R) DAAL|2020 Gold|[rls/daal-2020-rls](https://github.com/intel/daal/tree/rls/daal-2020-rls)</br>[rls/daal-2020-mnt](https://github.com/intel/daal/tree/rls/daal-2020-mnt) (contains ongoing fixes)|&nbsp;&nbsp;&nbsp;[Home page](https://software.intel.com/en-us/daal)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[Developer Guide](https://software.intel.com/en-us/daal-programming-guide)&nbsp;&nbsp;&nbsp;|
 
-Visit [oneDAL home page](https://software.intel.com/en-us/oneapi/onedal) for more information.
+## Table of Contents
 
-## Documentation
+- [Preview Features](#preview-features)
+- [Installation](#installation)
+- [Examples](#examples)
+- [How to Contribute](#how-to-contribute)
+- [Documentation](#documentation)
+- [Python API](#python-api)
+
+## Preview Features
+
+Preview features are introduced to gain early feedback from developers. A preview feature is subject to change in the future releases. Using a preview feature in a production code base is therefore strongly discouraged.
 
-Browse [oneDAL documentation](http://intel.github.io/daal/) on GitHub* or refer to the complete list of classical Intel(R) DAAL features on the official [Intel(R) DAAL website](https://software.intel.com/en-us/daal).
+- The `releases` branch contains the latest stable version of the library. The API for this branch can be considered stable.
+- The `master` branch contains preview features along with the stable ones. The stability of API for this branch is not guaranteed.   
+
+Users who wish to use, test, and provide feedback on the new features are encouraged to use the master branch. 
+
+---
+**Note:** The list of features that should be considered preview features is empty at this moment. When we add features with no guarantee of stability, features that reserve the right to change and break the API at any time, we will list them here.
+
+---
 
 ## Installation
 
 You can install oneDAL: 
 
-- from [oneDAL home page](https://software.intel.com/en-us/oneapi/onedal) as a part of Intel(R) Base Toolkit.
+- from [oneDAL home page](https://software.intel.com/en-us/oneapi/onedal) as a part of Intel(R) oneAPI Base Toolkit.
 - from [GitHub\*](https://github.com/intel/daal/releases).
 
 See [Installation from Sources](INSTALL.md) for details.
 
+## Examples
+
+Examples for different programming languages:
+
+- [C++](https://github.com/intel/daal/tree/master/examples/cpp)
+- [C++ SYCL*](https://github.com/intel/daal/tree/master/examples/cpp_sycl)
+- [Java](https://github.com/intel/daal/tree/master/examples/java)
+
+Data Examples for different computation modes:
+
+- [Batch](https://github.com/intel/daal/tree/master/examples/data/batch)
+- [Distributed](https://github.com/intel/daal/tree/master/examples/data/distributed)
+- [Online](https://github.com/intel/daal/tree/master/examples/data/online)
+
 ## How to Contribute
```

Comment:
Let's move this entire section to CONTRIBUTING.md file and just create link to it from readme . we can place contributing.md file in .github folder so it will not consume space, but still will be recognized by github

Reply:
Yes in general. but as there no rush with this change and assuming that change is pretty simple - i thought that we can simply update this PR.  It's up to @outoftardis  to decide on this.

Reply:
@napetrov separate file created, readme just links to it